.travis.yml

```diff
@@ -1,10 +1,16 @@
 language: objective-c
 osx_image: xcode8
-env: # Specify two projects here, so that each is run using the `script` specified below
-  - PROJECT_DIR=SampleProject
-  - PROJECT_DIR=SwiftSampleProject
-  - PROJECT_DIR="Paging Data Source Example"
-script: cd "${PROJECT_DIR}" && bundle exec fastlane test
+env:
+  global:
+    - SCAN_SKIP_BUILD=1
+  matrix:
+    # Define the details for each test that should be run
+    - SCAN_WORKSPACE="SampleProject/SampleProject.xcworkspace"                           SCAN_SCHEME="VOKCoreDataManager"         SCAN_SDK="iphonesimulator"
+    - SCAN_WORKSPACE="SampleProject/SampleProject.xcworkspace"                           SCAN_SCHEME="VOKCoreDataManager-tvOS"    SCAN_SDK="appletvsimulator"
+    - SCAN_WORKSPACE="SampleProject/SampleProject.xcworkspace"                           SCAN_SCHEME="VOKCoreDataManager-OSX"     SCAN_SDK="macosx"
+    - SCAN_WORKSPACE="SwiftSampleProject/SwiftSampleProject.xcworkspace"                 SCAN_SCHEME="SwiftyVokoder"          SCAN_SDK="macosx"
+    - SCAN_WORKSPACE="Paging Data Source Example/Paging Data Source Example.xcworkspace" SCAN_SCHEME="Paging Data Source Example" SCAN_SDK="iphonesimulator"
+script: bundle exec scan
```

Comment:
If you're just using `scan` now, can `fastlane` and/or `slather` be removed from the `Gemfile` and `Gemfile.lock`?